scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -1481,11 +1521,6 @@ def start_service(self, service, idx):
     fid = fetch_fid(self, service, idx)
     if fid == -1:
         return -1
-    #Run log rotate in background to avoid delay in startup
-    cmd = "/opt/seagate/cortx/motr/libexec/m0trace_logrotate.sh &"
```

Comment:
Why this is removed ?

Reply:
Actually, we are running  m0trace_logrotate.s and m0addb_logrotate.sh hourly as a part of cron jobs. So not needed to explicitly call it.